scripts/kb_scripts/copy_kb.py

```diff
@@ -10,7 +10,13 @@ def copy_kb(kb_path: str, copy_path: str) -> None:
         for i in os.listdir(copy_path):
             if os.path.isdir(os.path.join(copy_path, i)) and i == 'scripts':
                 shutil.rmtree(os.path.join(copy_path, i))
-            elif os.path.isfile(os.path.join(copy_path, i)):
-                os.remove(os.path.join(copy_path, i))
+            else:
+                list_of_files = list()
+                for (dir_path, dir_names, file_names) in os.walk(kb_path):
+                    list_of_files += [os.path.join(dir_path, file) for file in file_names]
+                for file in list_of_files:
+                    if not file.endswith('.scs') or not file.endswith('.scsi'):
```

Comment:
What about resources files(.png, .jpg and e.t.c). Why we can't copy all files from kb folders?